vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1361,7 +1349,16 @@ void vtkF3DRenderer::ConfigureTextActors()
     }
   }
 
-  this->UIActor->SetFontScale(this->FontScale);
+  double scaleFactor = 1.0;
+
+#ifdef _WIN32
+  const int dpi = GetDeviceCaps(wglGetCurrentDC(), LOGPIXELSY); // Default return 96
+  scaleFactor = static_cast<double>(dpi) / 96;
+#endif
+
+  const double adjustedFontScale = this->FontScale * scaleFactor;
+
+  this->UIActor->SetFontScale(adjustedFontScale);
```

Comment:
what if the user define the scaling themselves? Is it overwritten by the user value?